packages/uma-auth-client/src/steps/ConnectUma.tsx

```diff
@@ -53,7 +55,8 @@ export const ConnectUma = () => {
       const expirationPeriod = "year";
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line +51]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->These will also probably need to be injected into the oauth flow fwiw.<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/uma-universal-money-address/uma-js-sdk/122?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->